commercetools-internal-processors/src/test/java/io/sphere/sdk/annotations/processors/generators/UpdateActionsGeneratorTest.java

```diff
@@ -0,0 +1,28 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import io.sphere.sdk.annotations.processors.generators.examples.MyResource;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+public class UpdateActionsGeneratorTest extends AbstractMultipleGeneratorTest {
+
+    @Before
+    public void setup() {
+        generator = new UpdateActionsGenerator(compilationRule.getElements());
+    }
+
+    @Test
+    public void createMultipleUpdateActions() throws Exception {
+        final List<String> contentList = generateAsStrings(MyResource.class);
+        final List<String> expectedContent = expectedContent(MyResource.class);
+        int index = 0;
```

Comment:
I would add an `assertThat(contentlist.size()).isEquaTo(expectedContent.size())` here. This makes it clear what we expect.